### PR TITLE
Add Base support

### LINKED ISSUE #1: Add BASE chain
Base has recently launched and Aave V3 was deployed there. Need to add support for it.

### PR DESCRIPTION
**Pull request by NandyBa**
This pull request adds support of Base Aave V3

### CODE REVIEW

File: aavealarm/network.tsx [hunk 1]
```diff
@@ -103,6 +104,10 @@ function getAaveMarket(
       market = markets.AaveV3Metis;
       chainId = ChainId.metis_andromeda;
       break;
+    case chain == Chain.BASE && aaveVersion == 3:
+        market = markets.AaveV3Base;
```
Comment (thegodfatherbeeearth) [inline aavealarm/network.tsx:108]: I guess it should be `markets.AaveV3Basenet` here?

File: aavealarm/network.tsx [hunk 2]
```diff
@@ -103,6 +104,10 @@ function getAaveMarket(
...
+        chainId = ChainId.base;
```
Comment (thegodfatherbeeearth) [inline aavealarm/network.tsx:109]: When I tried it locally, I saw an error that `ChainId.base` doesn't exist. Did I do anything wrong?

File: data/rpc_nodes.json [hunk 3]
```diff
@@ -6,5 +6,6 @@
   "AVALANCHE": "https://avalancherpc.com",
   "ARBITRUM": "https://arb1.arbitrum.io/rpc",
   "OPTIMISM": "https://mainnet.optimism.io",
-  "METIS": "https://andromeda.metis.io/?owner=1088"
+  "METIS": "https://andromeda.metis.io/?owner=1088",
+  "BASE": "https://mainnet.base.org",
```
Comment (thegodfatherbeeearth) [inline data/rpc_nodes.json]:
```suggestion
  "BASE": "https://mainnet.base.org"
```

Invalid json

**Review by thegodfatherbeeearth — COMMENTED** [review]
Looks great! Left a couple of comments, @NandyBa

File: aavealarm/network.tsx (hunk 1 above)
Comment (NandyBa) [reply aavealarm/network.tsx:108]: https://github.com/bgd-labs/aave-address-book/blob/50b9acc87adc355d669895bdfc884a79fd4b9abe/src/ts/AaveAddressBook.ts#L28C13-L28C25 mention and export AaveV3Base

Change log [1.31.7](https://github.com/bgd-labs/aave-address-book/compare/v1.31.6...v1.31.7) change chain name from Basenet to Base so I assume Base was the correct name

File: aavealarm/network.tsx (hunk 2 above)
Comment (NandyBa) [reply aavealarm/network.tsx:109]: Got this on my side:

https://github.com/nebolax/aavealarm/assets/11545946/4dd9d815-9398-4e09-9bbf-b29843683364

File: aavealarm/network.tsx (hunk 1 above)
Comment (thegodfatherbeeearth) [reply aavealarm/network.tsx:108]: Sounds quite reasonable! But then `package.json` should be updated. Currently it contains `"@bgd-labs/aave-address-book": "^1.31.4"` (an older version).

File: aavealarm/network.tsx (hunk 2 above)
Comment (thegodfatherbeeearth) [reply aavealarm/network.tsx:109]: And I suppose `@aave/contract-helpers` should be updated in `package.json` too.

File: aavealarm/network.tsx (hunk 1 above)
Comment (NandyBa) [reply aavealarm/network.tsx:108]: Package have been updated

File: aavealarm/network.tsx (hunk 2 above)
Comment (NandyBa) [reply aavealarm/network.tsx:109]: Package have been updated

**Comment by thegodfatherbeeearth** [comment]
Alright, looks good!

**Pull request merged**